### PR TITLE
Refresh README.

### PR DESCRIPTION
**Pull request by jkodroff**

### CODE REVIEW

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
This is really great to read, so much clearer than what's there now! 🎉 

I had a couple of minor suggestions.

I think overall, if we could make it super clear that "`provider-ci` is the subfolder where all the magic happens" that would be awesome.

Formatting things:

I believe that the intersprinkling of code blocks with ``` (what did you call them?) is messing with the automatic numbering of a numbered list, so either let's use a bulleted list or number explicitly.

I'm not prepared to have the "two blank spaces after a period" argument but I'll note here that I noticed. 👀

**Review by infin8x — COMMENTED** [review]
Great to see this! Didn't approve since I'm not the right technical reviewer, but wanted to drop and note and say thanks for improving these docs!

File: README.md [hunk 1]
```diff
@@ -1,103 +1,112 @@
...
 
-```
-provider: rancher2
-env:
-  RANCHER_INSECURE: true
-  AWS_REGION: us-west-2
-```
+  ```bash
+  # Change the value of PROVIDER_NAME below:
+  PROVIDER_NAME=foo && mkdir providers/${PROVIDER_NAME} && touch providers/${PROVIDER_NAME}/config.yaml
```
[the remark this answers is not in the document]
Comment (jkodroff) [reply README.md]: Fixed - marking resolved.

File: README.md [hunk 2]
```diff
@@ -1,103 +1,112 @@
...
+Before generating any workflow files, run the following commands:
 
-First, ensure the dependencies are installed:
+1. Ensure the dependencies are installed:
 
-```
-$ npm install
-``` 
+  ```bash
+  cd provider && npm install
```
[the remark this answers is not in the document]
Comment (jkodroff) [reply README.md]: Yes.  Fixed - marking resolved.

File: README.md [hunk 3]
```diff
@@ -1,103 +1,112 @@
...
 
-* provider-ci
-* mp
-* infra
+* `provider-ci/` contains code to generate GitHub Actions workflow files for Pulumi providers, as well as the generated output for each provider (retained for the purpose of convenient output diffing).
+* `mp/` contains code using [microplane](https://github.com/Clever/microplane) to deploy the generated files in `provider-ci` to the many provider repositories.
+* `infra/providers/` contains a Pulumi program which uses [Pulumi GitHub provider](https://www.pulumi.com/registry/packages/github/) to ensure consistent [branch protections](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) across our provider repositories.
 
-## Provider CI
+  Because the code in this folder is a standard Pulumi program, we do not document its usage.  Refer to [the Pulumi docs](https://www.pulumi.com/docs/) for an overview if necessary.
```
[the remark this answers is not in the document]
Comment (jkodroff) [reply README.md]: Yeah, on second reading I must've been in a bad mood or something when I wrote that.  Suggestion accepted - marking resolved.

File: README.md [hunk 4]
```diff
@@ -1,103 +1,112 @@
 # Pulumi CI Management
 
-This repository is a 1 stop shop for the enablement of CI / CD process for the Pulumi provider ecosystem. The list of providers
-this repository covers can be found in provider. The repository has 3 parts to it:
+This repository contains code to manage CI/CD for the many Pulumi providers in a consistent and (mostly) automated manner.  The repo's intended audience are Pulumi Corp engineers, but its contents may also serve as a helpful example for Pulumi community members looking to maintain their own providers with a similar CI/CD process to Pulumi Corp.  The full list of providers this repository covers can be found in `provider-ci/providers`.  This repository has the following components:
 
-* provider-ci
-* mp
-* infra
+* `provider-ci/` contains code to generate GitHub Actions workflow files for Pulumi providers, as well as the generated output for each provider (retained for the purpose of convenient output diffing).
```
[the remark this answers is not in the document]
Comment (jkodroff) [reply README.md]: Fixed - marking resolved.

File: README.md [hunk 5]
```diff
@@ -1,103 +1,112 @@
...
+* `mp/` contains code using [microplane](https://github.com/Clever/microplane) to deploy the generated files in `provider-ci` to the many provider repositories.
+* `infra/providers/` contains a Pulumi program which uses [Pulumi GitHub provider](https://www.pulumi.com/registry/packages/github/) to ensure consistent [branch protections](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) across our provider repositories.
```
[the remark this answers is not in the document]
Comment (jkodroff) [reply README.md]: Fixed - marking resolved.

**Review by guineveresaenger — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**